lib/tldr.js

```diff
@@ -194,12 +194,13 @@ function renderContent(content, options={}) {
 
 function checkStale() {
   cache.lastUpdated((err, stats) => {
+    const cacheIsOutOfDate = stats.mtime < Date.now() - ms('30d');
     if (err) {
       console.error(err);
       exit(1);
     }
-    if (stats.mtime < Date.now() - ms('30d')) {
-      console.warn('Cache is out of date, you should run "tldr --update"');
+    if (cacheIsOutOfDate) {
+      exports.updateCache();
```

Comment:
Use the `cache.update()` call here. It is a bit unidiomatic to use the exported call directly.